data/1.1/jakarta-data-1.1.0-M1.html

```diff
@@ -0,0 +1,5456 @@
+<li><a href="#_ordering">5.4.6. Ordering</a></li>
+<li><a href="#_logical_operators">5.4.7. Logical operators</a></li>
+</ul>
+</li>
+<li><a href="#_clauses">5.5. Clauses</a>
+<ul class="sectlevel3">
+<li><a href="#_from_clause">5.5.1. From clause</a></li>
+<li><a href="#_where_clause">5.5.2. Where clause</a></li>
+<li><a href="#_select_clause">5.5.3. Select clause</a></li>
+<li><a href="#_set_clause">5.5.4. Set clause</a></li>
+<li><a href="#_order_clause">5.5.5. Order clause</a></li>
+</ul>
+</li>
+<li><a href="#_statements">5.6. Statements</a>
+<ul class="sectlevel3">
+<li><a href="#_select_statements">5.6.1. Select statements</a></li>
+<li><a href="#_update_statements">5.6.2. Update statements</a></li>
+<li><a href="#_delete_statements">5.6.3. Delete statements</a></li>
+</ul>
+</li>
+<li><a href="#_syntax">5.7. Syntax</a></li>
+</ul>
+</li>
+<li><a href="#_jakarta_data_providers">6. Jakarta Data providers</a>
+<ul class="sectlevel2">
+<li><a href="#_providers_and_repositories">6.1. Providers and repositories</a></li>
+<li><a href="#_provider_support_for_entities">6.2. Provider support for entities</a></li>
+<li><a href="#_provider_name">6.3. Provider name</a></li>
+</ul>
+</li>
+<li><a href="#_interoperability_with_other_jakarta_ee_specifications">7. Interoperability with other Jakarta EE specifications</a>
+<ul class="sectlevel2">
+<li><a href="#_jakarta_contexts_and_dependency_injection">7.1. Jakarta Contexts and Dependency Injection</a>
+<ul class="sectlevel3">
+<li><a href="#_cdi_dependency_injection">7.1.1. CDI dependency injection</a></li>
+<li><a href="#_cdi_events">7.1.2. CDI events</a></li>
+<li><a href="#_cdi_extensions_for_jakarta_data_providers">7.1.3. CDI extensions for Jakarta Data providers</a></li>
+</ul>
+</li>
+<li><a href="#_jakarta_interceptors">7.2. Jakarta Interceptors</a></li>
+<li><a href="#_jakarta_transactions">7.3. Jakarta Transactions</a></li>
+<li><a href="#_jakarta_persistence">7.4. Jakarta Persistence</a></li>
+<li><a href="#_jakarta_nosql">7.5. Jakarta NoSQL</a></li>
+<li><a href="#_jakarta_bean_validation">7.6. Jakarta Bean Validation</a></li>
+</ul>
+</li>
+<li><a href="#_portability_in_jakarta_data">8. Portability in Jakarta Data</a>
+<ul class="sectlevel2">
+<li><a href="#_portability_for_relational_databases">8.1. Portability for relational databases</a></li>
+<li><a href="#_portability_for_nosql_databases">8.2. Portability for NoSQL databases</a>
+<ul class="sectlevel3">
+<li><a href="#_key_value_databases">8.2.1. Key-value databases</a></li>
+<li><a href="#_wide_column_databases">8.2.2. Wide-column databases</a></li>
+<li><a href="#_document_databases">8.2.3. Document databases</a></li>
+<li><a href="#_graph_databases">8.2.4. Graph databases</a></li>
+</ul>
+</li>
+</ul>
+</li>
+</ul>
+</div>
+</div>
+<div id="content">
+<div id="preamble">
+<div class="sectionbody">
+<div class="literalblock">
+<div class="content">
+<pre>Specification: Jakarta Data
+
+Version: 1.1.0-M1
```

Comment:
This (and above and in other places) perhaps should be `1.1-M1` for specification.